### ISSUE TITLE
Crash when renaming pane

### ISSUE THREAD

**Issue by jakobnissen** [zellij-org/zellij#937]
How to reproduce:
* Enter zellij with `zellij`
* Enter pane mode: Ctrl-p
* Rename: `c`
* Zellij crashes with error message
```
Error occurred in server:
Originating Thread(s):
1. stdin_handler_thread: AcceptInput
2. plugin_thread: Update

Error: thread 'wasm' panicked at 'called `Result::unwrap()` on an `Err` value: RuntimeError { source: Trap(UnreachableCodeReached), wasm_trace: [FrameInfo { module_name: "<module>", func_index: 127, function_name: None, func_start: SourceLoc(79296), instr: SourceLoc(82825) }, FrameInfo { module_name: "<module>", func_index: 72, function_name: None, func_start: SourceLoc(44826), instr: SourceLoc(45013) }, FrameInfo { module_name: "<module>", func_index: 71, function_name: None, func_start: SourceLoc(44772), instr: SourceLoc(44820) }, FrameInfo { module_name: "<module>", func_index: 54, function_name: None, func_start: SourceLoc(37454), instr: SourceLoc(37575) }, FrameInfo { module_name: "<module>", func_index: 267, function_name: None, func_start: SourceLoc(151119), instr: SourceLoc(167920) }], native_trace:    0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
 }': zellij-server/src/wasm_vm.rs:146
   0: backtrace::backtrace::trace
   1: backtrace::capture::Backtrace::new
   2: zellij_utils::errors::handle_panic
   3: std::panicking::rust_panic_with_hook
   4: std::panicking::begin_panic_handler::{{closure}}
   5: std::sys_common::backtrace::__rust_end_short_backtrace
   6: _rust_begin_unwind
   7: core::panicking::panic_fmt
   8: core::result::unwrap_failed
   9: zellij_server::wasm_vm::wasm_thread_main
  10: std::sys_common::backtrace::__rust_begin_short_backtrace
  11: core::ops::function::FnOnce::call_once{{vtable.shim}}
  12: std::sys::unix::thread::Thread::new::thread_start
  13: __pthread_start
```

**Basic information**
Zellij version: 0.22.0 (unreleased master branch, commit 448b805)
`uname -av`: `Darwin Jakobs-MacBook-Pro.local 21.1.0 Darwin Kernel Version 21.1.0: Wed Oct 13 17:33:23 PDT 2021; root:xnu-8019.41.5~1/RELEASE_X86_64 x86_64`
Alacritty version: 0.9.0

**Comment by imsnif** [comment]
@jakobnissen - coming to think of it, this might have been an issue involving plugins not being updated properly in the development version. I just released version 0.22.0, could you install it and give it a try? (I'm personally not reproducing this)

**Comment by jakobnissen** [comment]
Upgrading to v 0.22.1 fixed the issue - thanks for the quick fix! Closing as solved.
